.travis/build.sh

```diff
@@ -5,6 +5,7 @@ set -o errexit
 export CFLAGS="-Werror"
 export CGO_CFLAGS="-DCI_BUILD"
 
+make precheck
```

Comment:
Couldn't we just run `make`?

Reply:
Good point, changed in the latest revision. I was a bit worried that we might need additional build dependencies installed. But from a quick look everything seems to be there and otherwise we'll see from the run on this PR :smile:

Reply:
I figured since Travis CI is the only CI piece that tests aarch64, checking that it compiles there is probably worth the extra build time.

Reply:
Might be worth using `-j2` though. Travis CI usually runs on 2 cores and some part of the builds benefit from that.